examples/digits_dann_lightn/tutorial.ipynb

```diff
@@ -38,10 +38,10 @@
     "if 'google.colab' in str(get_ipython()):\n",
     "    print('Running on CoLab')\n",
     "    !pip uninstall --yes imgaug && pip uninstall --yes albumentations && pip install git+https://github.com/aleju/imgaug.git\n",
-    "    !pip install git+https://github.com/pykale/pykale.git\n",
-    "    \n",
     "    !git clone https://github.com/pykale/pykale.git\n",
-    "    %cd pykale/examples/digits_dann_lightn\n",
+    "    %cd pykale\n",
+    "    !pip install .[medim,example] \n",
```

Comment:
Looks weird/confusing to install medical imaging for digits. Shall we change `medim` to `vision`/`image` (since now it has `skimage` and not just dicom)? We can discuss this afternoon.

Reply:
Yes. Alternatively, we can create another API for medical image related functions only.